pypeit/par/pypeitpar.py

```diff
@@ -1563,7 +1636,7 @@ def valid_spectrographs():
         # parameter sets (like DetectorPar) and where they go needs to
         # be rethought.
         return ['keck_deimos', 'keck_lris_blue', 'keck_lris_red', 'keck_lris_red_longonly',
-                'keck_nires', 'keck_nirspec_low', 'keck_mosfire', 'keck_hires_red',
+                'keck_nires', 'keck_nirspec_low', 'keck_mosfire', 'keck_hires_red', 'keck_kcwi_blue',
```

Comment:
Do we need to define "blue"?  I.e., could we just use `keck_kcwi` (and then `keck_kcrm` when it comes online)?

Reply:
The "red" version has a different name (KCRM), so I think we can get away with just `keck_kcwi`, but I'm not too fussed.